src/Morphic-Widgets-Taskbar/TaskbarMorph.class.st

```diff
@@ -432,8 +432,10 @@ TaskbarMorph >> updateTasks [
 	wm := self windows.
 	taskbarTasksFromWorldMorph := (wm collect: [ :m | m taskbarTask ])
 		                              reject: [ :task |
-		                              task isNil or: [
-			                              task morph visible not ] ].
+			                              task isNil or: [
+				                              SpClosedWindowListPresenter 
```

Comment:
This introduces a dependency from Morph to Spec which is not right. 

Maybe we could add the property to the morph? 

You can use Morph>>#setProperty:toValue: to keep the state in the morph

Reply:
Or maybe the Taskbar could store the almost closed windows in a var without depending of Spec?